File: code-coaches/medium/roadrunner/Roadrunner.swift
func isRoadrunnerSafe(distanceToSafety: Int, roadrunnerSpeed: Int, coyoteSpeed: Int) -> Bool {
    let secondsToSafety = Double(distanceToSafety) / Double(roadrunnerSpeed)
    let roadrunnerDistance = Double(roadrunnerSpeed) * secondsToSafety
    let coyoteDistance = Double(coyoteSpeed) * secondsToSafety - 50
    return roadrunnerDistance > coyoteDistance
}

func main() {
    let distanceToSafety = Int(readLine()!)!
    let roadrunnerSpeed = Int(readLine()!)!
    let coyoteSpeed = Int(readLine()!)!

	if isRoadrunnerSafe(distanceToSafety: distanceToSafety, roadrunnerSpeed: roadrunnerSpeed, coyoteSpeed: coyoteSpeed) {
		print("Meep Meep")
	} else {
		print("Yum")
	}
}

main()

File: code-coaches/medium/roadrunner/Roadrunner.cs
using System;

public class Roadrunner
{
    static bool IsRoadrunnerSafe(int distanceToSafety, int roadrunnerSpeed, int coyoteSpeed)
    {
        double secondsToSafety = distanceToSafety / roadrunnerSpeed;
        double roadrunnerDistance = roadrunnerSpeed * secondsToSafety;
        double coyoteDistance = coyoteSpeed * secondsToSafety - 50;
        return roadrunnerDistance > coyoteDistance;
    }

    static void Main(string[] args)
    {
        int distanceToSafety, roadrunnerSpeed, coyoteSpeed;
        distanceToSafety = int.Parse(Console.ReadLine());
        roadrunnerSpeed = int.Parse(Console.ReadLine());
        coyoteSpeed = int.Parse(Console.ReadLine());

        if (IsRoadrunnerSafe(distanceToSafety, roadrunnerSpeed, coyoteSpeed))
            Console.WriteLine("Meep Meep");
        else
            Console.WriteLine("Yum");
    }
}
